src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -53,23 +103,43 @@ const CpsOnwardJourney = ({
   };
   const { script, service, dir } = useContext(ServiceContext);
   const CpsOnwardJourneyWrapper = ({ children }) =>
-    enableGridWrapper ? (
-      <GridWrapper {...a11yAttributes}>
-        <Wrapper>{children}</Wrapper>
-      </GridWrapper>
+    parentColumns ? (
+      <Wrapper parentColumns={parentColumns} {...a11yAttributes}>
+        {children}
+      </Wrapper>
     ) : (
-      <Wrapper {...a11yAttributes}>{children}</Wrapper>
+      <LegacyGridWrapper {...a11yAttributes}>
+        <LegacyGridItemConstrainedLarge>
+          {children}
+        </LegacyGridItemConstrainedLarge>
+      </LegacyGridWrapper>
     );
+
   CpsOnwardJourneyWrapper.propTypes = {
     children: node.isRequired,
   };
+
   if (!content.length) return null;
   const hasSingleContent = content.length === 1;
   const [singleContent] = content;
 
+  const WrapperItem = ({ children }) => (
+    <Wrapper parentColumns={constrainedLargeGridColumns} item>
+      {children}
+    </Wrapper>
+  );
+
+  WrapperItem.propTypes = {
+    children: node.isRequired,
+  };
+
+  const CpsOnwardJourneyItemWrapper = parentColumns
+    ? WrapperItem
+    : LegacyGridItemConstrainedLarge;
+
   return (
     <CpsOnwardJourneyWrapper>
-      <Wrapper>
+      <CpsOnwardJourneyItemWrapper>
```

Comment:
I don't think we need this item wrapper anymore as it's adding extra padding which we don't need, removal of this wrapper will get rid of the excess padding. It looks like this also fixes the issue you have with IE11 that @Simon2828 has found.

Reply:
Good spot @HarveyPeachey , yeah we should make this change as we do need to support IE11 @Simon2828 (I think as a team we are now able to choose to deliver a degraded experience if the complexity of supporting IE11 is too much, not really the case here I'd say)

Reply:
Looks good on IE11 on browserstack now 👍

Reply:
Also looks good on firefox both map and STY